DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Reports/_ActivityTable.cshtml

```diff
@@ -1,40 +1,45 @@
 ﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.Reports
-
 @model UsageStatsTableViewModel
 
 <table role="table" class="nhsuk-table-responsive">
   <thead role="rowgroup" class="nhsuk-table__head">
-    <tr role="row">
-      <th role="columnheader" scope="col">
-        Period
-      </th>
-      <th role="columnheader" scope="col">
-        Registrations
-      </th>
-      <th role="columnheader" scope="col">
-        Completions
-      </th>
-      <th role="columnheader" scope="col">
-        Evaluations
-      </th>
-    </tr>
+  <tr role="row">
+    <th role="columnheader" scope="col">
+      Period
+    </th>
+    <th role="columnheader" scope="col">
+      Registrations
+    </th>
+    <th role="columnheader" scope="col">
+      Completions
+    </th>
+    <th role="columnheader" scope="col">
+      Evaluations
+    </th>
+  </tr>
   </thead>
   <tbody class="nhsuk-table__body">
-  @foreach (var activityRow in Model.Rows) {
-    <tr role="row" class="nhsuk-table__row">
-      <td role="cell" class="nhsuk-table__cell">
-        <span class="nhsuk-table-responsive__heading">Period </span>@activityRow.Period
-      </td>
-      <td role="cell" class="nhsuk-table__cell">
-        <span class="nhsuk-table-responsive__heading">Registrations </span>@activityRow.Registrations
-      </td>
-      <td role="cell" class="nhsuk-table__cell">
-        <span class="nhsuk-table-responsive__heading">Completions </span>@activityRow.Completions
-      </td>
-      <td role="cell" class="nhsuk-table__cell">
-        <span class="nhsuk-table-responsive__heading">Evaluations </span>@activityRow.Evaluations
-      </td>
-    </tr>
+  @{
+    var rowCount = 0;
+    foreach (var activityRow in Model.Rows) {
+      rowCount++;
+      <tr role="row" class="nhsuk-table__row @(rowCount > 5 ? "js-hidden-activity-row" : string.Empty)">
+        <td role="cell" class="nhsuk-table__cell">
+          <span class="nhsuk-table-responsive__heading">Period </span>@activityRow.Period
+        </td>
+        <td role="cell" class="nhsuk-table__cell">
+          <span class="nhsuk-table-responsive__heading">Registrations </span>@activityRow.Registrations
+        </td>
+        <td role="cell" class="nhsuk-table__cell">
+          <span class="nhsuk-table-responsive__heading">Completions </span>@activityRow.Completions
+        </td>
+        <td role="cell" class="nhsuk-table__cell">
+          <span class="nhsuk-table-responsive__heading">Evaluations </span>@activityRow.Evaluations
+        </td>
+      </tr>
+    }
   }
   </tbody>
 </table>
+
+<a class="nhsuk-u-margin-top-4 js-only-block  js-shown-load-more" href="#">View More</a>

```

Comment:
Should this be a button instead, so people using a screen reader won't think it takes them to a new page?

Reply:
I have changed it to button but do we need to style it to be shown as a link, or can I leave it as a button as shown below?

![image](https://user-images.githubusercontent.com/90783892/136254576-73b1aaae-5a0a-4bb3-a901-f209756b0b34.png)

Reply:
Thanks. I have changed it back to link with role being button.